topics/computational-chemistry/tutorials/analysis-md-simulations/tutorial.md

```diff
@@ -0,0 +1,192 @@
+- The **PCA visualization** {% icon tool %} tool will carry out PCA and return a trajectory of the selected principle component. This trajectory is useful for visualisation and further investigating the interesting modes and changes that occur within a selected principle component. 
+
+
+> ### {% icon hands_on %} Hands-on: Calculate PCA
+>
+> 1. **PCA** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"dcd trajectory input"*: Trajectory file
+>    - {% icon param-file %} *"pdb input"*: Structure file
+>    - *"Use singular value decomposition (SVD) instead of default eigenvalue decomposition ?"*: `No`
+>    - *"Select domains"*: `Calpha`
+>
+> 1. **PCA visualization** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"dcd trajectory input"*: Trajectory file
+>    - {% icon param-file %} *"pdb input"*: Structure file
+>    - *"Use singular value decomposition (SVD) instead of default eigenvalue decomposition ?"*: `No`
+>    - *"Select domains"*: `Calpha`
+>    - *"Principal component id"*: `Calpha`
+>
+> PCA visualisation: This tool can generate small trajectories of the first three principal components. The .pdb of the .nc files can be visualized using a visualization software such as VMD.
+>
+{: .hands_on}
+
+![Snapshot of PCA plot]({{ site.baseurl }}{% link topics/computational-chemistry/images/PCA_plot_46.png %} "PCA plot for a short CBH1 simulation")
+
+> > ### {% icon question %} Question
+> >
+> > What do the features in the RMSD plot tell us? Do the principal coordinates have a meaning?
+> >
+> > > ### {% icon solution %} Solution
+> > > Here, PCA shows the statistically meaningful conformations in the CBH1 trajectory. The principal motions within the trajectory and the vital motions needed for conformational changes can be identified. Two distinct groupings along the PC1 plane, indicating a non-periodic conformational change, are identified. The groupings along the PC2 and PC3 planes do not completely cluster separately, implying that these global motions are periodic. The PC1 is linked to an active site motion that limits the motion to a key glycosidic bond.
+> > {: .solution}
+> {: .question}
+
+## Workflow vs. individual tools
+You can choose to use the tools one by one as described above, or alternatively combine into a single analysis using the workflow provided.
+![Snapshot of conformational analysis workflow]({{ site.baseurl }}{% link topics/computational-chemistry/images/workflow_confanalysis.png %} "A simple analysis workflow")
+
+Have a look at the published workflows (and histories) by navigating to 'Shared Data' in the upper tab on usegalaxy.eu and in the dropdown menu selecting 'Workflows'. Search for 'Bio3d' in the search box and choose 'MD Analysis using Bio3D'.
```

Comment:
We try to avoid any reference to a specific Galaxy instance.
Maybe here, you could have an hand-on box to show how to import the workflow from the training material into Galaxy. What do you think?